db/migrate/20170707145338_remove_columns_from_attendance.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveColumnsFromAttendance < ActiveRecord::Migration[5.1]
```

Comment:
Not necessary to change anything here. Just wanted to note that the indentation looks a bit off 👮 

Maybe check your editor config ✅